CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Here's what we have in our C++ project for that we want to use staccato:
```
cmake_minimum_required(VERSION 3.13)
```
Let's use the same version for CMake here.